resource-managers/mesos/pom.xml

```diff
@@ -49,6 +49,13 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-tags_${scala.binary.version}</artifactId>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
```

Comment:
This is needed to fix the following error due to `org.apache.spark.tags.ExtendedYarnTest`.
```
[info] Running Spark tests using Maven with these arguments:  -Phadoop-2.3 -Phive -Pyarn -Pmesos -Phive-thriftserver -Pkinesis-asl -Dtest.exclude.tags=org.apache.spark.tags.ExtendedHiveTest,org.apache.spark.tags.ExtendedYarnTest test --fail-at-end
...
[INFO] Spark Project Mesos ................................ FAILURE [ 10.687 s]
...
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project spark-mesos_2.11: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test failed: There was an error in the forked process
[ERROR] java.lang.RuntimeException: Unable to load category: org.apache.spark.tags.ExtendedYarnTest
[ERROR] at org.apache.maven.surefire.group.match.SingleGroupMatcher.loadGroupClasses(SingleGroupMatcher.java:139)
[ERROR] at ...
[ERROR] Caused by: java.lang.ClassNotFoundException: org.apache.spark.tags.ExtendedYarnTest
...
```